internal/config/cgmgr/stats_linux.go

```diff
@@ -238,3 +257,89 @@ func isMemoryUnlimited(v uint64) bool {
 	// either way, libcontainer/cgroups will return math.MaxUint64
 	return v == math.MaxUint64
 }
+
+func cgroupPidStats(stats *cgroups.Stats, pids []int) *PidsStats {
+	var fdCount, socketCount, ulimitsSoft uint64
+
+	// This is based on the cadvisor handler: https://github.com/google/cadvisor/blob/master/container/libcontainer/handler.go
+	for _, pid := range pids {
+		addFdsForProcess(pid, &fdCount, &socketCount)
+		addUlimitsForProcess(pid, &ulimitsSoft)
+	}
+
+	return &PidsStats{
+		Current:         stats.PidsStats.Current,
+		Limit:           stats.PidsStats.Limit,
+		Pids:            pids,
+		FileDescriptors: fdCount,
+		Sockets:         socketCount,
+		Threads:         stats.PidsStats.Current,
+		ThreadsMax:      stats.PidsStats.Limit,
```

Comment:
We removed the original logic to obtain the threads in cbea275. Is that correct? Now we count the number of PIDs, not the threads.